src/codemodder/codemods/xml_transformer.py

```diff
@@ -171,12 +171,13 @@ def apply(
         file_context: FileContext,
         results: list[Result] | None,
     ) -> ChangeSet | None:
-        if file_context.file_path.suffix.lower() not in (".config", ".xml"):
+        if (file_path := file_context.file_path).suffix.lower() not in (
```

Comment:
Actually isn't this already handled by `default_extensions` on the codemod itself?